File: Java implementation/src/Constant.java
import java.util.HashSet;
import java.util.Set;

public class Constant implements Arithmetic {

    private int n;

    public Constant(int n) {
        this.n = n;
    }

    @Override
    public Set<String> fv() {
        return new HashSet<String>();
    }

    @Override
    public String toString() {
        return Integer.toString(n);
    }


}

File: CppUpdated/Graph/BasicActions/Arithmetics/Constant.cpp

#include "Constant.h"
#include <vector>

Constant::Constant(int value){
    this->value = value;
}

string Constant::getType() {
    return "constant";
}

Identifiers Constant::getUsedVariables() {
    return Identifiers();
}

string Constant::toString() {
    return to_string(this->value);
}